src/scripts/main.js

```diff
@@ -1 +1,15 @@
 'use strict';
+
+const gallery = document.querySelector('.gallery');
+const list = document.querySelector('.gallery__list');
+
+list.addEventListener('click', (events) => {
+  const item = events.target;
+  const largeImg = gallery.children[0];
+  const itemCopy = item.cloneNode(true);
+
+  events.preventDefault();
+  largeImg.firstElementChild.remove();
+  itemCopy.className = 'gallery__large-img gallery__img';
+  largeImg.append(itemCopy);
+});
```

Comment:
```suggestion
const gallery = document.querySelector('.gallery');
const list = document.querySelector('.gallery__list');

list.addEventListener('click', (event) => {
  const item = events.target;
  const largeImg = gallery.children[0];
  const itemCopy = item.cloneNode(true);

  events.preventDefault();
  largeImg.firstElementChild.remove();
  itemCopy.className = 'gallery__large-img gallery__img';
  largeImg.append(itemCopy);
});
```
You can just change href of img tag. Let's try to solve it using href attribute.

Reply:
I did not change name variable 'events' on 'event' because show error: ''event' is already declared in the upper scope'